.github/workflows/tests.yml

```diff
@@ -229,6 +228,13 @@ jobs:
           bundle exec rails r bin/monitor-manifest
           bundle exec rails test:prepare
 
+      ###
+      # Wait for everything to be ready
+      - name: Wait for open search to become ready
+        env:
+          OPENSEARCH_HOST: https://127.0.0.1:${{ job.services.opensearch.ports['9200'] }}
+        run: ./bin/wait_for_opensearch
```

Comment:
I wonder at what step we should be having this check. We're not doing a bunch of other things first that will be meaningless if opensearch won't start